src/MSALWrapper/PCACache/IPCACache.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.MSALWrapper.AuthFlows
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// The PCACache interface.
+    /// </summary>
+    internal interface IPCACache

```

Comment:
Not a blocker, but more of an open question that may only resolve itself as we include other auth types. Is this cache actually only usable for public client flows or would it also be suitable for confidential client ones as well? 🤔

Reply:
Kyle and I were looking into confidential client top level and did see cache. I will have better answers as I deep dive into each Auth flow. 

@kyle-rader do you know top of your head?